packs/sf2e/heritages/ysoki/deep-ysoki.json

```
@@ -0,0 +1,32 @@
+{
+    "_id": "aUEsp1nn5ZimPKE3",
+    "folder": "P1wu8p0R9xU2jqgy",
```

```suggestion
    "folder": "bB5HzELH07UIY03Z",
```
We'll need to update the folder ID for the heritages folder too